### ISSUE TITLE
When I compile the FasterTransformer on P40, I set DSM to 60 for cmake. However, when I make, the custom_ar_kernels.cu error 

### ISSUE THREAD

**Issue by grainw** [NVIDIA/FasterTransformer#250]
### Description

```shell
master ,pytorch:21.11-py3n,P40
```

### Reproduced Steps

```shell
nvidia-docker run -ti --rm nvcr.io/nvidia/pytorch:21.11-py3 bash
git clone https://github.com/NVIDIA/FasterTransformer.git
mkdir -p FasterTransformer/build
cd FasterTransformer/build
git submodule init && git submodule update
cmake -DSM=61 -DCMAKE_BUILD_TYPE=Release -DBUILD_PYT=ON -DBUILD_MULTI_GPU=ON ..
make
Building CUDA object src/fastertransformer/kernels/CMakeFiles/custom_ar_kernels.dir/custom_ar_kernels.cu.o
ptxas /tmp/tmpxft_000039cf_00000000-6_custom_ar_kernels.ptx, line 603; error   : Feature '.release' requires .target sm_70 or higher
ptxas /tmp/tmpxft_000039cf_00000000-6_custom_ar_kernels.ptx, line 614; error   : Feature '.acquire' requires .target sm_70 or higher
ptxas /tmp/tmpxft_000039cf_00000000-6_custom_ar_kernels.ptx, line 4583; error   : Feature '.release' requires .target sm_70 or higher
ptxas /tmp/tmpxft_000039cf_00000000-6_custom_ar_kernels.ptx, line 4594; error   : Feature '.acquire' requires .target sm_70 or higher
ptxas /tmp/tmpxft_000039cf_00000000-6_custom_ar_kernels.ptx, line 5367; error   : Feature '.release' requires .target sm_70 or higher
ptxas /tmp/tmpxft_000039cf_00000000-6_custom_ar_kernels.ptx, line 5378; error   : Feature '.acquire' requires .target sm_70 or higher
ptxas fatal   : Ptx assembly aborted due to errors
src/fastertransformer/kernels/CMakeFiles/custom_ar_kernels.dir/build.make:75: recipe for target 'src/fastertransformer/kernels/CMakeFiles/custom_ar_kernels.dir/custom_ar_kernels.cu.o' failed
make[2]: *** [src/fastertransformer/kernels/CMakeFiles/custom_ar_kernels.dir/custom_ar_kernels.cu.o] Error 255
CMakeFiles/Makefile2:2920: recipe for target 'src/fastertransformer/kernels/CMakeFiles/custom_ar_kernels.dir/all' failed
make[1]: *** [src/fastertransformer/kernels/CMakeFiles/custom_ar_kernels.dir/all] Error 2
Makefile:135: recipe for target 'all' failed
make: *** [all] Error 2
```

**Comment by PerkzZheng** [comment]
Issue solved in this pr (https://github.com/NVIDIA/FasterTransformer/pull/251).
let us know if you meet other problems.

**Comment by grainw** [comment]
Thanks for your reply, my problem is solved, thank you very much

**Comment by grainw** [comment]
> Thanks for your reply, my problem is solved, thank you very much



> Issue solved in this pr (#251). let us know if you meet other problems.

Thanks for your reply, my problem is solved, thank you very much
